README.md

```diff
@@ -68,18 +68,16 @@ As a developer, there are few steps to building your own secure federated learni
 You can use KotlinSyft as a front-end or as a background service. The following is a quick start example usage:
 
 ```kotlin
-    val userId = "my Id"
-
     // Optional: Make an http request to your server to get an authentication token
-    val authToken = apiClient.requestToken("https://www.mywebsite.com/request-token/$userId")
```

Comment:
`apiClient.requestToken()` does not appear in this library, so I removed it. Maybe it comes from a dependency?

Reply:
Good catch. This has changed and now it is the responsibility of the client to implement the authentication. The demo app uses a hard-coded value for it.